### PR TITLE
Fix export dialog title for generic pipelines

### LINKED ISSUE #2319: Export pipeline title for generic pipelines
**Describe the issue**
The export dialog for generic pipelines currently has the title `Export pipeline for Generic`:
image
Although the additional text used for context (`for x`) should only be applied for runtime specific pipelines (kfp, airflow).

**Expected behavior**
Either drop the additional text for generic pipelines (original behavior prior to runtime types), or reword it to something like `Export generic pipeline`

**Deployment information**
Describe what you've deployed and how:
 - Elyra version: v3.4.0-dev
 - Operating system: macos
 - Installation source: source

### PR DESCRIPTION
**Pull request by karlaspuldaro**
Fixes #2319 
Update the dialog title used to export generic pipelines, which currently reads `Export pipeline for Generic`.

### What changes were proposed in this pull request?
Dropped the additional export title for generic pipelines
image

For runtime specific pipelines, the additional context is still used
image

### How was this pull request tested?
Added integration test for export dialog title.
(unrelated to this PR) Uncommented integration tests for runtime info displayed in the VPE toolbar.
 
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

**Review by kevin-bates — APPROVED** [review]
LGTM - thanks @karlaspuldaro!

**Review by ajbozarth — APPROVED** [review]
Tested locally and this LGTM
